index.html

```diff
@@ -4,6 +4,10 @@
   <title>YOUR NAME HERE</title>
 </head>
 <body>
+  <h1>Adam Reid</h1>
+  <img>https://media.giphy.com/media/gU25raLP4pUu4/giphy.gif</>
```

Comment:
uh oh, did you mean to change both files?  Or did you intend to copy index.html to dam.html ?